components/file_upload/file_upload.tsx

```diff
@@ -347,9 +361,9 @@ export class FileUpload extends PureComponent {
             }
 
             // For IE browsers
-            if (types.contains && !types.contains('Files')) {
-                return;
-            }
+            // if (types.contains && !types.contains('Files')) {
```

Comment:
Typescript was complaining about this part. Because `.contains` doesn't exist on `string[]`. 

Considering that IE browser is not widely used anymore and supported by Microsoft, I would remove it. 

Please let me know what do you think?

Reply:
Removed, thanks